src/target/riscv/riscv.c

```diff
@@ -172,6 +172,8 @@ struct scan_field select_idcode = {
 
 
 int bscan_tunnel_ir_width; /* if zero, then tunneling is not present/active */
+int bscan_tunnel_type; /* if zero, then nested tap tunneling 1 : data register tunnel */
```

Comment:
Please make this an enum, e.g. something like:
enum {
   BSCAN_TUNNEL_NESTED_TAP,
   BSCAN_TUNNEL_DATA_REGISTER
}  bscan_tunnel_type;

Reply:
Resolved with new type bscan_tunnel_type_t in riscv.h . All references to type 0 / type 1 have been changed appropriately.